web/src/main/java/org/cbioportal/weblegacy/mixin/DataAccessTokenMixin.java

```diff
@@ -0,0 +1,16 @@
+package org.cbioportal.web.mixin;
```

Comment:
do we really need DataAccessTokenMixin.java file? There isn't any web legacy services use DataAccessToken right?

Reply:
@khzhu this might simply be in the wrong package. I'll move it

Reply:
@khzhu this might simply be in the wrong package... I'll move